### PR TITLE
fix: MUI Theme Base CSS Styles

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝
- Oh no! Our MUI theme isn't installed correctly! 😖
- The background color of the body is white ⚪️ in dark mode and it drives me absolutely nuts 🌰

## Major Changes 🔄
- Set missing UI theme styles
- Add `<CssBaseline />` to mount base styles

### Before ☹️

https://github.com/linode/manager/assets/115251059/b3088fa1-0e78-484d-8dab-a52049db4ea4

### After 😁

https://github.com/linode/manager/assets/115251059/6e9aa519-b56e-48ce-9a07-38931fe054f9

## How to test 🧪
- Checkout this PR
- Check that the body's background in light mode is white
- Check that the body's background is in dark mode is gray
- Check for general style regressions

### CODE REVIEW

File: packages/manager/src/components/PrimaryNav/PrimaryNav.styles.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ const useStyles = makeStyles<void, 'linkItem'>()(
       color: '#fff',
       display: 'flex',
       fontFamily: 'LatoWebBold', // we keep this bold at all times
+      fontSize: '1rem',
```
Comment (bnussman-akamai) [inline packages/manager/src/components/PrimaryNav/PrimaryNav.styles.ts:45]: Fixes fontSize regression

File: packages/manager/src/features/Footer/Footer.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ interface Props {
 const useStyles = makeStyles()((theme: Theme) => ({
   container: {
     backgroundColor: theme.bg.main,
+    fontSize: '1rem',
```
Comment (bnussman-akamai) [inline packages/manager/src/features/Footer/Footer.tsx:17]: Fixes fontSize regression

**Review by cpathipa — APPROVED** [review]
TY @bnussman-akamai for fixing this.

**Review by abailly-akamai — CHANGES_REQUESTED** [review]
Thanks for working on this!

This is a PR that needs far more visual testing before approving (CC @cpathipa). We really need to pull the code and test in the browser until we have visual regression tools. I tested quickly and found regressions right away so I suspect there could be more.

Screenshot 2023-09-06 at 16 19 53

Screenshot 2023-09-06 at 16 18 50

**Comment by bnussman-akamai** [comment]
@abailly-akamai 

1. First regression was fixed by merging `develop` in (https://github.com/linode/manager/pull/9622)
2. Second screenshot is fixed by 990e8a1

**Comment by mjac0bs** [comment]
Tbh, I can't see a different in font size in Alban's screenshot [990e8a1](https://github.com/linode/manager/pull/9636/commits/990e8a1dd72d20ea6d840e88226fa226505a62d4) fixed and thought that table arrow was pointing to the typo in "Transferred". It looked wrong spelled as is, so I googled it. Two "r"s.  (I'm sorry 😅 )

**Comment by bnussman-akamai** [comment]
Wait I'm confused... @abailly-akamai, what is the first screenschot suppose to show?

**Comment by abailly-akamai** [comment]
@bnussman-akamai 🤣  the grid cells were misaligned

**Comment by bnussman-akamai** [comment]
Ah okay, my fault. Should be good then! (https://github.com/linode/manager/pull/9622)

**Review by abailly-akamai — APPROVED** [review]
Thanks for the fixes! Looks good - went through most flows and couldn't find aby other visual regressions 👍

**Pull request merged**